hack/ansible-lint.sh

```diff
@@ -17,33 +17,35 @@
 # under the License.
 #
 #~Usage:
-#~  ansible-lint.sh [-f] [-p PROFILE]
+#~  ansible-lint.sh [-p PROFILE] [PATH]
 #~
 #~Options:
-#~  -c      Remove .linted markers
-#~  -p      Use the "production" ansible-lint profile
-#~  -n      Don't clear screen after linting each role
+#~  -b BIN  Path to the `ansible-lint` binary (default: $PATH/ansible-lint)
+#~  -d      Download requirements declared in galaxy.yml (default: False)
+#~  -p      Use the "production" ansible-lint profile (default: shared)
+#~Arguments:
+#~  PATH    (Optional) path to lint, defaults to the top level directory
 
 function usage(){
     grep "^#~" "$0" | sed -e "s/#~//"
 }
 
 TOPDIR="$(git rev-parse --show-toplevel)"
 CWD="$PWD"
-CLEAN=""
 PROFILE="shared"
-CLEAR="yes"
+BIN="$(which ansible-lint)"
+OFFLINE="--offline"
 
-while getopts ":cpn" o; do
+while getopts ":b:dp" o; do
     case $o in
-        c)
-            CLEAN="yes"
-            ;;
         p)
             PROFILE="production"
             ;;
-        n)
-            CLEAR=""
+        b)
+            BINARY="$OPTARG"
```

Comment:
Should this be BIN instead of BINARY?  Otherwise I don't see how the which check below will work?